### DISCUSSION TITLE
How to refresh Birdseye and Camera view

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#3046] [Ask A Question]
One of my cameras went offline 2 days ago but Birdseye and Cameras view still shows image from before it went offline.
Had to restart Frigate to clear the image before resetting the camera.

Am I missing a setting? If not it would be useful; when connection is lost to show an "OFFLINE" icon instead of last image taken.

**Comment by blakeblackshear** [comment]
There is an existing feature request for this already.

**Reply by 18923** [reply]
FYI, Dead camera still showing up in Birdseye and Camera View using Latest Beta (Current version: 0.11.0-beta4)

**Reply by 18923** [reply]
Is there an ETA when this may be fixed?

**Reply by NickM-27** [reply]
Not in 0.11, maybe in 0.12

**Reply by 18923** [reply]
Thank you for the update.

**Reply by 18923** [reply]
FYI, Birdseye still showing last image taken from disconnected camera. Using latest beta (version: 0.13-beta5)

**Reply by 18923** [reply]
FYI: I'm still experiencing this. Running 0.13.0-49814B3

**Reply by MRACHINI** [reply]
Hello,

I'm still facing this issue, whenever a camera goes offline it has the last image stuck in birdseye
